README.md

```diff
@@ -36,3 +36,12 @@ cargo clippy
 ```
 
 Alas, `cargo build` doesn't work. That's a known problem and we're working on it.  Please use `cargo make build` instead for now.
+
+### Setting up self-hosted CI docker container
+
+To set up docker container for CI, you can run the following command:
+
+```sh
+docker build -t ceno-runner scripts/ci/
+docker run -d ceno-runner
```

Comment:
I can execute these docker commands locally, but I don't think that's going to change the container that GitHub actually uses?  Or does it?

Reply:
Can you leave some remarks in the ReadMe about who's responsible for our runners, and how to reach them, when eg something breaks or we want to upgrade?

Reply:
You have to click the  button `add new runner` explicitly.

Reply:
Ok, makes sense.  But I doubt you are running that on your laptop?

Could you add just a tiny bit of context about what you have to do?  Presumably you contact someone in Scroll to help you get access to a machine or something like that?